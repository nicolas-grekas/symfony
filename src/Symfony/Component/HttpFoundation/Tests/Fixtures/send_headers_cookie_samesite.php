<?php

use Symfony\Component\HttpFoundation\Cookie;
use Symfony\Component\HttpFoundation\Response;

require __DIR__ . '/common.inc';

$r = new Response();
$r->headers->set('Date', 'Thu, 02 Feb 2012 12:12:12 GMT');
$r->headers->setCookie(new Cookie('SameSite', 'Strict'));
$r->sendHeaders();
