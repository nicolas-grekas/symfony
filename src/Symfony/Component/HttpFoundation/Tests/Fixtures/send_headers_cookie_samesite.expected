
Array
(
    [0] => Content-Type: text/plain; charset=utf-8
    [1] => Cache-Control: no-cache, private
    [2] => Date: Thu, 02 Feb 2012 12:12:12 GMT
    [3] => Set-Cookie: SameSite=Strict; path=/; httponly
)
shutdown
